### PR TITLE
initial output plugin

### PR DESCRIPTION
**Pull request by andrewctate**
Adds tooling, output plugin class, a placeholder feed function, and tests.

This lays the groundwork for porting the actual DCAT-US logic from Indexer.

### CODE REVIEW

File: LICENSE [hunk 1]
```diff
@@ -0,0 +1,201 @@
...
+      To apply the Apache License to your work, attach the following
+      boilerplate notice, with the fields enclosed by brackets "{}"
+      replaced with your own identifying information. (Don't include
+      the brackets!)  The text should be enclosed in the appropriate
+      comment syntax for the file format. We also recommend that a
+      file or class name and description of purpose be included on the
+      same "printed page" as the copyright notice for easier
+      identification within third-party archives.
+
+   Copyright {yyyy} {name of copyright owner}
```
Comment (rgwozdz) [inline LICENSE:189]: I think you need to add year and Owner.

File: src/dcat-us/index.test.ts [hunk 2]
```diff
@@ -0,0 +1,110 @@
...
+    'source-934d07d3f163470ab9125e585a0f59fe',
+    'source-undefined',
+    'Web Map',
+    'Registered App',
+  ],
+  description:
+    'Create your own initiative by combining existing applications with a custom site. Use this initiative to form teams around a problem and invite your community to participate.',
+  tags: ['Hub Site'],
+  snippet:
+    'I love front end development and everyone I\'ve met as Esri so far has been really nice and pleasant and I am excited to get to know everyone better yes',
```
Comment (rgwozdz) [inline src/dcat-us/index.test.ts:47]: Nice to hear this.

File: src/index.test.ts [hunk 3]
```diff
@@ -0,0 +1,139 @@
...
+    app = express();
+    app.get('/dcat', plugin.serve.bind(plugin));
+
+    return [ plugin, app ];
+  }
+
+  beforeEach(() => {
+    jest.resetModules();
+
+    // const {
```
Comment (rgwozdz) [inline src/index.test.ts:33]: Do we still need this commented out code?

**Review by rgwozdz — APPROVED** [review]

File: src/dcat-us/index.ts [hunk 4]
```diff
@@ -0,0 +1,21 @@
+import { IItem } from '@esri/arcgis-rest-portal';
+import { FeedFormatterStream } from './feed-formatter-stream';
+
+export function getDataStreamDcatUs11(siteItem: IItem) {
+  console.log(siteItem);
```
Comment (sonofflynn89) [inline src/dcat-us/index.ts:5]: Just a friendly reminder to remove this when you can 😁

**Review by sonofflynn89 — APPROVED** [review]
Good job, man!

File: src/dcat-us/index.test.ts (hunk 2 above)
Comment (andrewctate) [reply src/dcat-us/index.test.ts:47]: Lol—that's from Jules.

File: src/index.test.ts (hunk 3 above)
Comment (andrewctate) [reply src/index.test.ts:33]: It's coming back in the next PR.

File: src/dcat-us/index.ts (hunk 4 above)
Comment (andrewctate) [reply src/dcat-us/index.ts:5]: Thank you! Yeah, just my attempt to get around Typescript's checks.

File: LICENSE (hunk 1 above)
Comment (andrewctate) [reply LICENSE]: Good catch

**Pull request merged**